src/views/DownloadPackedOrders.vue

```diff
@@ -8,7 +8,7 @@
     </ion-header>
 
     <ion-content>
-      <main>
+      <main v-if="content.length > 0">
```

Comment:
As we have added parent level check for content.length, remove all checks for content.length inside this parent.